packages/gatsby-theme-carbon/gatsby-ssr.js

```diff
@@ -9,7 +9,7 @@ export const wrapRootElement = wrapRoot;
 export const onRenderBody = ({ setHeadComponents, setBodyAttributes }) => {
   // prevents jank while page hydrates, removed on page load
   setBodyAttributes({
-    class: 'body--preload',
+    className: 'body--preload',
```

Comment:
Gah! There it was! For the life of me.. I could not find where that error was coming from.